packages/api/src/image-registry.ts

```diff
@@ -0,0 +1,29 @@
+/**********************************************************************
+ * Copyright (C) 2024 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+export interface ImageSearchOptions {
+  registry?: string;
+  query: string;
+  limit?: number;
+}
+
+export interface ImageSearchResult {
+  name: string;
+  description: string;
+  star_count: number;
+  is_official: boolean;
```

Comment:
Why are you using underscore separation ? would `isOfficial` and `starCount` be consistent with _most_ of the code base ?

Reply:
To match with what is returned by the API call, so `return JSON.parse(resultJSON.body).results` works without any mapping

Reply:
okey 👍